### DISCUSSION TITLE
Fetching data on Expo environment

### DISCUSSION THREAD

**Discussion by Pelema** [supabase/supabase#9670] [Questions] [answered]
Hi I'm trying to fetch data from supabase via react expo but I'm not getting any data back do you perhaps know what the issue is or can you guide me maybe I'm misunderstanding some concepts. The same code works on reactJS

**Comment by GaryAustin1** [comment, marked as answer]
You probably need to provide more data for someone to help you.  If you are getting empty arrays back usually you have a filter or RLS problem (user not setup when you think it is).   But you are not clear on if there is an error, or even what you are doing.

**Reply by Pelema** [reply]
Okay cool, that fixed it
